### ISSUE TITLE
How to configure istio to open port 80 / 443?

### ISSUE THREAD

**Issue by catman002** [istio/istio#32931]
[ ] Docs
[ ] Installation
[x ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure
[ ] Upgrade

**istio1.19.5**

How to configure istio to open port 80 / 443?
I set istio ingress gateway (80) and gateway (80), it doesn't work. 
Don't I have permission?

**Comment by catman002** [comment]
**edit istio-ingressgateway:**
 ISTIO_META_UNPRIVILEGED_POD=true was set, removed that env var.
 then changed the securityContext like this:
···
        securityContext: 
          privileged: true
          readOnlyRootFilesystem: true
···

**Comment by jmajoor** [comment]
Where / how do you set these configurations?
```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  meshConfig:
    discoverySelectors:
      - matchLabels:
          istio-discovery: enabled
    accessLogFile: /dev/stdout
    defaultConfig:
      proxyMetadata:
        ISTIO_META_DNS_CAPTURE: "false"
  values:
    global:
      logging:
        level: 'all:debug'
      meshID: mesh1
      multiCluster:
        clusterName: east-cluster
      network: network1
    gateways:
      istio-ingressgateway:
        runAsRoot: true
        env:
          ISTIO_META_UNPRIVILEGED_POD: "false"
```
However I still get the  ` skipping privileged gateway port 443 for node istio-ingressgateway-559d9c4ccf-vsr4h.istio-system as it is an unprivileged pod` in the log and no listeners for the ingress gateway.

**Comment by ronyrv13** [comment]
Hello @jmajoor 

Is the issue resolved for you?..
We are facing the same issue for port 587.

Tried all ways to resolve error -
`skipping privileged gateway port 587 for node istio-ingressgateway as it is an unprivileged pod`

Raised an issue - https://github.com/istio/istio/issues/38295

Thought to ask you before updating the issue!

**Comment by jmajoor** [comment]
@ronyrv13 
We never figured out how to deal with the unprivileged pod error. We did get things working in the end.

In our case our pods are actually listening on 8443/8080 and we have a K8S service that is defined on 443/80.

We then configured a gateway on 8443 with a virtual service. 
The virtual service defines the K8s service as the host with the 8080 port (ie. the backend port defined in our K8s service).
My guess is that the virtual service just looks at the K8S service's endpoints.

The configuration did create 443/80 listeners in the load balancer, which then are going in to the cluster. Based on hostname routing it then finds our gateway and virtual service.

So in the end it all works by avoiding the unprivileged pod.
